tuf/client/client.go

```diff
@@ -422,7 +420,8 @@ func (c *Client) downloadTargets(role string) error {
 }
 
 func (c *Client) downloadSigned(role string, size int64, expectedSha256 []byte) ([]byte, *data.Signed, error) {
-	raw, err := c.remote.GetMeta(role, size)
+	rolePath := utils.URLFilePath(role, expectedSha256)
```

Comment:
Yeah, I read what it does, I just think that something like `CanonicalPath` is less confusing than it including URL in the name.